apps/dashboard/src/components/side-navigation/changelog-cards.tsx

```diff
@@ -0,0 +1,207 @@
+import { useState, useEffect } from 'react';
+import { motion } from 'motion/react';
+import { RiCloseLine } from 'react-icons/ri';
+import { useQuery } from '@tanstack/react-query';
+import { useTelemetry } from '@/hooks/use-telemetry';
+import { TelemetryEvent } from '@/utils/telemetry';
+
+type Changelog = {
+  id: string;
+  date: string;
+  title: string;
+  notes?: string;
+  version: number;
+  imageUrl?: string;
+  published: boolean;
+};
+
+type CachedChangelogs = {
+  data: Changelog[];
+  timestamp: number;
+};
+
+const CONSTANTS = {
+  CHANGELOG_API_URL: 'https://productlane.com/api/v1/changelogs/f13f1996-c9b0-4fea-8ee7-2c3faf6a832d',
+  CACHE_KEY: 'cached_changelogs',
+  CACHE_EXPIRY: 4 * 60 * 60 * 1000, // 4 hours
+  NUMBER_OF_CARDS: 3,
+  DISMISSED_STORAGE_KEY: 'dismissed_changelogs',
+  CARD_OFFSET: 10,
+  SCALE_FACTOR: 0.06,
+} as const;
+
+export function ChangelogStack() {
+  const [changelogs, setChangelogs] = useState<Changelog[]>([]);
+  const track = useTelemetry();
+
+  const fetchChangelogs = async (): Promise<Changelog[]> => {
+    const cachedData = storage.getCachedData();
+    if (cachedData) {
+      return filterChangelogs(cachedData, storage.getDismissedChangelogs());
+    }
+
+    const response = await fetch(CONSTANTS.CHANGELOG_API_URL);
+    const rawData = await response.json();
```

Comment:
do we have any interface of this data?

Reply:
I want to cache it between reloads, so we don't always make api calls to product lane on each page load as this info rarely changes. But, good point